www/_posts/2021-08-21-removal-of-translated-docs.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Notice of the Removal of the Translated Docs"
+categories: announcements
+tags: news
+---
+
+We like to announce that we have deprecated and removed all translated docs.
```

Comment:
"We like" -> "We'd like"

Reply:
But also, please consider giving this some time; first, deprecate, **then** remove.

Reply:
I changed it to `We'd` as suggested but, I am making a note that we are removing. We are not talking about `would like to remove`. So the tone of this is wrong and lying.

Reply:
I changed it to`We'd` as suggested.

Reply:
Thanks!